pkg/interfaces/contracts/standalone-utils/IProtocolFeeBurner.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IProtocolFeeBurner {
+    event ProtocolFeeBurned(
```

Comment:
Not sure if we want to document this event, or if the documentation of the `burn()` function is enough

Reply:
It depends on the burner design - this would be emitted by the burner, not the sweeper, since burning is asynchronous. It depends on whether the burner will internally track and retry ones that fail (maybe ultimately just forwarding them to the recipient). Or whether the tokens will be sent back to the sweeper (and perhaps recovered directly, in which case we'd want to know the amounts in the event, etc.) There are a lot of ways we could design this. Could also be tracked off-chain with events (e.g., by matching up nonces, as noted in the description).